docs/developer/architecture/logs-components-diagram.uml

```diff
@@ -0,0 +1,49 @@
+@startuml
+
+node "node1" {
+    journal_logs1 <-- [collector1] : read
```

Comment:
I'm not expert either, I asked myself for the direction of the arrow (my understanding is from where comes the action, e.g. a client querying a database is client -> db, but I know you can also represent that with client <-> db, because the client is asking and the db is returning the data), I wasn't sure how to do it, that's why I added the "read" for it to be clear.
I like the `reads from` / `writes to`, it's even clearer.